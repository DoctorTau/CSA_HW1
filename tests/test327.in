43
-4363 -2570 7523 -2166 4311 -8495 9249 7093 9465 -2670 4054 -5024 -6736 -4363 -8644 8046 5575 -9942 6970 -4989 3007 2005 3362 -6694 -1947 -5814 -9571 8003 2183 -7801 -8040 204 -5896 8076 -1620 -8013 -57 -284 3057 -1949 -5114 -6895 5725 