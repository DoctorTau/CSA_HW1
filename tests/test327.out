-2570 -2166 -2670 -5024 -6736 -8644 -9942 -6694 -5814 -8040 -5896 -1620 -284 -5114 